components/k8s-model-server/http-proxy/server.py

```diff
@@ -86,7 +122,12 @@ def decode_b64_if_needed(value):
     else:
         return value
 
+
 class PredictHandler(tornado.web.RequestHandler):
+    """
+    Predict Hanlder proxy predict method, the input of tf savedModel is expected to be a `Map<strinbg, tf.Tensor>` protobuf
```

Comment:
What does protobuf mean in this context? i.e. does it mean the json representation of the protobuf?

Reply:
the protobuf is referring to backend.. from the user perspective, the input schema is the same